jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -14,7 +14,7 @@ lib = library(identifier: 'jenkins@6.3.2', retriever: modernSCM([
 ]))
 
 def docker_images = [
-    'tar': 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3',
+    'tar': 'opensearchstaging/ci-runner:ci-runner-almalinux8-opensearch-build-v1',
```

Comment:
I believe this is generic image? With 1.13.16 going on, any idea how to handle 1x release pipeline?

Reply:
Sounds good. Still think its high time to start maintaining per version docker images even for integTest considering it is one of the main blockers now during release. As of now, if we have 2.13.1, the integTest results may vary since the underlying docker image changed. However, for tar I believe we are still picking from the manifest? https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.13.0/opensearch-2.13.0-test.yml#L6

Reply:
Yes.